lms/djangoapps/grades/tests/test_models.py

```diff
@@ -267,7 +267,7 @@ def test_update(self):
         self.assertEqual(created_grade.id, updated_grade.id)
 
     def test_create(self):
-        self.assertTrue(len(PersistentCourseGrade.objects.values()) == 0)
+        self.assertEquals(len(PersistentCourseGrade.objects.values()), 0)
```

Comment:
Nit: assertEquals is deprecated, and doesn't exist in python 3.  Use assertEqual instead.